shared/js/content-scope/fingerprint.js

```diff
@@ -153,7 +153,7 @@ function initCanvasProtection (args) {
                     return target.apply(thisArg, args)
                 }
                 try {
-                    const { offScreenCanvas } = computeOffScreenCanvas(thisArg.canvas)
```

Comment:
I think this got changed out in a rebase, without this `toDataURL` and `toBlob` aren't protected.